package.json

```diff
@@ -57,6 +57,7 @@
     "typescript": "^5.1.3"
   },
   "dependencies": {
+    "@types/tape": "^5.6.4",
```

Comment:
@creilly8 I think this should be a devDependency, otherwise codebases that uses `@sjcrh/proteinpaint-client`, such as the GFF repo, will end up installing tape types even though it will not be used in that codebase.